### PR TITLE
Adding selinux-testsuite tests

### PR DESCRIPTION
**Pull request by nasastry**
security/selinux-tests.py adds selinux-testsuite tests

Signed-off-by: Nageswara R Sastry <rnsastry@linux.vnet.ibm.com>

### CODE REVIEW

**Comment by nasastry** [comment]
```
Fetching asset from selinux-tests.py:SELinux.test
JOB ID     : 4f8c378d7ed85e6d75657eec09979fa97c5faa9c
JOB LOG    : /home/nasastry/avocado-fvt-wrapper/results/job-2020-09-08T15.14-4f8c378/job.log
 (1/1) selinux-tests.py:SELinux.test: FAIL: 6 test(s) failed, please refer to the log (117.02 s)
RESULTS    : PASS 0 | ERROR 0 | FAIL 1 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB HTML   : /home/nasastry/avocado-fvt-wrapper/results/job-2020-09-08T15.14-4f8c378/results.html
JOB TIME   : 119.47 s
```

File: security/selinux-tests.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+        archive.extract(tarball, self.workdir)
+        self.sourcedir = os.path.join(self.workdir, 'selinux-testsuite-master')
+        os.chdir(self.sourcedir)
+        output = process.system_output('getenforce', ignore_status=True)
+        if output.decode() == 'Permissive':
+            process.run('setenforce 1', ignore_status=True)
+            output = process.system_output('getenforce', ignore_status=True)
+            if output.decode() == 'Permissive':
+                self.fail("Unable to enter in to 'Enforcing' mode")
+        status, output = process.getstatusoutput('make -C policy load',
```
Comment (PraveenPenguin) [inline security/selinux-tests.py:71]: build.run_make can be used and we can use `<variable>.exit_status` and `<variable>.stdout` for next processing 

for more info refer below 

https://github.com/avocado-framework/avocado/blob/master/avocado/utils/build.py#L51

File: security/selinux-tests.py [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+                                                 verbose=True)
+        if status != 0:
+            self.fail("Failed to load the policies")
+
+    def test(self):
+        '''
+        Running tests from selinux-testsuite
+        '''
+        count = 0
+        output = process.system_output('make -C tests test', ignore_status=True,
```
Comment (PraveenPenguin) [inline security/selinux-tests.py:81]: here also build.run_make can  be used

File: security/selinux-tests.py [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+                         "lksctp-tools-devel", "attr", "libbpf-devel",
+                         "keyutils-libs-devel", "quota", "xfsprogs-devel",
+                         "libuuid-devel", "nftables", "kernel-devel",
+                         "kernel-modules", "perl-Test-Harness", "coreutils",
+                         "netlabel_tools", "libsepol", "checkpolicy",
+                         "libselinux", "policycoreutils", "libsemanage",
+                         "nfs-utils", "policycoreutils-newrole",
+                         "xfsprogs-devel", "libselinux-devel"])
+        else:
+            self.cancel("Install the package for perf supported\
```
Comment (PraveenPenguin) [inline security/selinux-tests.py:53]: string should be `selinux test suite  supported `

**Comment by nasastry** [comment]
```
# avocado run selinux-tests.py
Fetching asset from selinux-tests.py:SELinux.test
JOB ID     : e7b7555de4d08c0d3bb051e7d2ddd3159831a015
JOB LOG    : /home/nasastry/avocado-fvt-wrapper/results/job-2020-09-08T16.42-e7b7555/job.log
 (1/1) selinux-tests.py:SELinux.test: FAIL: 6 test(s) failed, please refer to the log (116.81 s)
RESULTS    : PASS 0 | ERROR 0 | FAIL 1 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB HTML   : /home/nasastry/avocado-fvt-wrapper/results/job-2020-09-08T16.42-e7b7555/results.html
JOB TIME   : 119.24 s
```

**Comment by nasastry** [comment]
@PraveenPenguin comments taken care. Please review. Thanks!!

**Review by PraveenPenguin — APPROVED** [review]
LGTM

**Pull request merged**